src/Suave/Http.fs

```diff
@@ -220,7 +220,7 @@ module Http =
      
 
     let not_modified : WebPart =
-      fun ctx -> { ctx with response = {status = HTTP_304; headers = []; content = Bytes [||] }} |> succeed
+      fun ctx -> { ctx with response = {status = HTTP_304; headers = []; content = Bytes [||]; writePreamble = true }} |> succeed
```

Comment:
Sometimes we do not want Suave to send the HTTP preamble. Like in the Xsp module, the module is going to write directly to the socket and we do not want the default Suave workflow to execute. It is a little bit of a hack.

I think the need also arised when @haf was working on Server-Sent events.